tt_metal/hw/firmware/src/idle_erisc.cc

```diff
@@ -152,10 +152,10 @@ int main() {
                 CLEAR_PREVIOUS_LAUNCH_MESSAGE_ENTRY_FOR_WATCHER();
             }
 
```

Comment:
Just finishing the profiler scope above the forever loop so profiling scope actually ends